CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ set(CMAKE_CXX_STANDARD 11)
 
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 
+# NOTE(compnerd) this is a horrible workaround for Windows to ensure that the
```

Comment:
Can you please do this for Windows only, and not for other platforms? I don't think having an identical directory layout between platforms is necessary; each have their own conventions which we should follow.

Reply:
This is the same thing that LLVM does on all the platforms, as does the Swift project.  What do we gain by doing it as a special case here?  Furthermore, this is the build tree only, so having the identical directory layout is necessary to keep the overall packaging rules identical.  (e.g. copying of files).

Reply:
Ah, right, this is for the build layout, not the installed layout.